src/pages/settings/Subscription/CardSection/BillingBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type BillingBannerProps = {
+    title: string;
+    subtitle: string;
+    isError?: boolean;
+    shouldShowRedDotIndicator?: boolean;
+};
+
+function BillingBanner({title, subtitle, isError, shouldShowRedDotIndicator}: BillingBannerProps) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    return (
+        <View style={[styles.pt4, styles.pb3, styles.ph5, styles.flexRow, styles.gap3, styles.w100, styles.alignItemsCenter, styles.hoveredComponentBG]}>
+            <Icon
+                src={isError ? Illustrations.CreditCardEyes : Illustrations.CheckmarkCircle}
```

Comment:
I think the icon should be passed as a prop. Because we're planning to reuse this for RBR from other API failures like [change of subscription plan](https://github.com/Expensify/App/pull/43029)

Reply:
or may be I am getting confused. This banner is specific to the Card section? and the other RBRs will be handled as different implementation?

Reply:
We do not need to pass here icon as a prop, for BRB pattern we will pass the `shouldShowRedDotIndicator` prop

Reply:
Yes, currently, this banner is specific

Reply:
I meant for all the errors it'll always show `CreditCardEyes`. Wouldn't we want to make this reusable?

Reply:
Fine with that.